climada/engine/unsequa/unc_output.py

```diff
@@ -424,9 +424,8 @@ def plot_sample(self, figsize=None):
         figsize : tuple(int or float, int or float), optional
             The figsize argument of matplotlib.pyplot.subplots()
             The default is derived from the total number of plots (nplots) as:
-
-            >>> nrows, ncols = int(np.ceil(nplots / 3)), min(nplots, 3)
-            >>> figsize = (ncols * FIG_W, nrows * FIG_H)
+                nrows, ncols = int(np.ceil(nplots / 3)), min(nplots, 3)
```

Comment:
Why the change in style?

Reply:
no idea. I reverted the change in style.